packages/rum-core/src/domain/rumEventsCollection/resource/resourceCollection.ts

```diff
@@ -48,6 +48,7 @@ function processRequest(request: RequestCompleteEvent) {
     {
       date: preferredTimeStamp(startClocks),
       resource: {
+        id: generateUUID(),
```

Comment:
Since now all resources have an id, you could remove the `resource.id` from the `computeRequestTracingInfo` function below.